ndicapimodule.cxx

```diff
@@ -1527,9 +1528,9 @@ ndicapiExport MOD_INIT(ndicapy)
 
   Py_NDIErrcodeMacro(NDI_ENVIRONMENT);
 
-  Py_NDIErrcodeMacro(NDI_EPROM_READ);
-  Py_NDIErrcodeMacro(NDI_EPROM_WRITE);
-  Py_NDIErrcodeMacro(NDI_EPROM_ERASE);
+  //Py_NDIErrcodeMacro(NDI_EPROM_READ);
```

Comment:
Do these no longer work? If so, could you add an issue?

If they do work, please undo the commenting.

Reply:
Removed due to missing symblols. I can protect them with an #ifdef